src/renderer/modules/injector.ts

```diff
@@ -280,11 +280,11 @@ export class Injector {
      * @returns Uninject Function.
      */
     addPopoverButton: (item: GetButtonItem) => {
-      const uninjector = window.replugged.api.messagePopover.addButton(item)
+      const uninjector = window.replugged.api.messagePopover.addButton(item);
```

Comment:
Import this function instead of calling it on the window

Reply:
The class has to be initialized... which it is in replugged.ts. Can't import it from there...

Reply:
Switch it to a namespace instead of class. That way it doesn't need to be initialized at all.

You could also initialize it within that file, but a namespace is much better for this.